docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+recover button with a label of ``Yes, I'm sure``.
+
+However if there are any conflicts that might be resolved by user (object
+model was registered with ``aldryn-reversions`` decorator AND a with admin
+mixin for model admin registration) you will see a list of links to recover
+related objects first.
+
+.. note::
+    In current implementation only required FKs are considered as a conflicts,
+    but this would be changed in next versions since if relation existed for
+    not required FK, and related object was deleted you will end up with
+    ``IntegrityError`` when you will try to restore this object.
+    In such cases if you know which related object is missing and if it was
+    registered with ``aldryn-reversion`` for revision tracking and with admin
+    mixin - you can recover related object first and then return to restore
+    this object.
+
+In case if related object was not registered with admin mixin but it was
+registered for revision tracking ``aldryn-reversion`` would try to solve
+conflict automatically by examining required FK relations.
+In case of success you might be able to restore the object and objects that
+are required relations to this object.
+
+.. note::
+    Be aware that automatic resolver would try to use selected object reivison
+    which means that related objects would be restored at version at which
+    they were in that revision (not the object latest version).
+    So be sure to examine related objects and edit them to the desired state.
+
+****************************
+Translations (django-parler)
+****************************
+
+If model that is registered with ``aldryn-reversion`` contains translatable
+fields each form (restore object version and recover deleted object) will
+also have a choice to select translations to restore.
+In that case at least one translation should be selected.
+
+*********************
+CMS placeholder field
+*********************
+
+If model has ``placeholder`` fields and ``aldryn-reversion`` was not
+configured to ignore those fields they will also be tracked as a part of
+object revision.
+That means that in case if placeholder object that represents model's
+placeholder field would be deleted - you will be notified that it will be
+restored (in case of recover process).
+
+.. note::
+    If only placeholder object was deleted (not the plugins that it holds)
+    then you will be able to get palceholder AND plugins, most likely
+    plugins were not changed (relation to placeholder).
+    But if palceholder AND related plugins were deleted then restoring
+    placeholder object won't restore the plugins, which means that it will
+    be just an empty placeholder. Though you might be able to restore plugins
+    as well if after recovering deleted object you will restore object version
+    which contains plugins as well.
+
+In case of reverting history it would be restored to selected objcet version
```

Comment:
"When reverting history, the object will be restored to the corresponding revision automatically, so it may be a good idea to check the restored object and its plugins and edit them where necessary."